src/openrct2/actions/MazeSetTrackAction.hpp

```diff
@@ -344,7 +344,7 @@ struct MazeSetTrackAction : public GameActionBase<GAME_COMMAND_SET_MAZE_TRACK, G
 
         map_invalidate_tile(floor2(_x, 32), floor2(_y, 32), tileElement->base_height * 8, tileElement->clearance_height * 8);
 
-        if ((tileElement->properties.track.maze_entry & 0x8888) == 0x8888)
+        if ((tileElement->AsTrack()->GetMazeEntry() & 0x8888) == 0x8888)
```

Comment:
Any clue as to what these magic numbers mean? (And is there a constant for them?)